### PR TITLE
Add task 3

### PR DESCRIPTION
**Pull request by MariusLang**
> [!IMPORTANT]
> If you stumbled upon this PR before the task was officially released and started working on it early, you will be disqualified if we notice! **Be fair to your fellow students!**

---

Adds the 3nd task while publishing the first one for everyone.

_This should not be merged before 15/12/24, 10 o'clock._

### CODE REVIEW

File: content/tasks/second/epilogue.md [hunk 1]
```diff
@@ -1 +1,8 @@
-AkASItlDu+UHt/vQT1YfD20ZfC43P88ugn5706JwDQWwZr83y3NzW3GpE2KCkK5FiShECyVLi8/fUDkE2NJxDco3BdsLc+DnZOvAu/RqfI2v70iD4SgOa+ghtPGcQy8u63yo5ACFbTpNtYEl6tEubg/JucwNN+ZDgzXsPfOaUitDDrBGW2DUOfQu67/8uxPAXk99TFGY0ezTqIuBz3ipjtEYeHZ6WRhcwOd9YmWHnC5wYqqlONXig4GfSNxwS2ltSRG8aMBZKDdoUQAngLTENce+eGIJ9D+aHKES6Q93YTvL+P9To/Y8lWuhRkX/efFQQP+UiA6RlGT4GmNyrhKcvl7PESiawmRT3iF+qnuTBUTKPtMRqzN36fr8NoftmfGFIQug4ulwtGrfzFbzNBWmggno5lgNO1jgaOsAsaoYCOuHAgwQcjO4QlZCRDB+NvvnUH7pCzWXaW+ddxQF6bLmsf046X11L3FXBPd3M6tW4FuOH637g4N0NwDyxF9yhxbeGvoUuqiYVezSkw76qj8BGBcks1LxEk5qWlUlG90wUhgPZeNomIHp6m3hAs0maYiTTKoDzSuGMQ0Aw0nlAC3iQQb/KHOJCZWj8L3U4/Nb1Z0iSHgmktDSbvlvL1I8I+cQcwgVUpAt+kBWbWzlVi6XDBHaCg8zxbap90QYh438DGiymmA=
\ No newline at end of file
+# Geschenkgrößen zählen
```
Comment (Kuschel-Swein) [inline content/tasks/second/epilogue.md:1]: This uses the wrong heading level compared to the task 1

File: content/tasks/third/index.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
+# Santacloud reparieren
```
Comment (Kuschel-Swein) [inline content/tasks/third/index.md:1]: This also uses the wrong heading level

File: content/tasks/third/epilogue.md [hunk 3]
```diff
@@ -0,0 +1 @@
+0Du4seztsw/2/6TZkoJdJgH3hjVK6ZD21EAun+BEBzczwcl6PMg7hYL2+KBqEyySKZV2NxrKQux4lFTacrVpa46WDi1qNObiMKftmPy/VVeV/c10lpIRnwBw1cM8xJC85eACzZaoALdCzPruINojOvTMhccI7hv4AztJ5D1Gaw17yLbJBRaxPNI7l61Y7K5I2S8HahT6LK5fAYJIUY7VZaYTV/jjap1A5NA5Wa32YjOWOfPLrqERdnG7l2NzQThjoLBXq7qOGPVy/MiOf71eeEhfk0tmVqkDv9RvG7LFplk277upYMF2hDE/Iayj6IAAzmQE4aPBYOeIKGIHc9HNMHBWj5rTJ8wk8VAgAiiMvudO2n6QtgYVlB2DnC/D+ZkcyOSZG5OL3KqMVXoVBO2LOaqopYwKzKYJjfO6ZtB1rTogb9CAZN2riMN1faTKMUS4COra5fJcyWsM0ScPULDluE+IjRbLcBZjztpxRBPiTklMRT3hgMOj0xmNcWr6vptloiRxa/Tb1mvmbgIU+GkzPBX5ytANx4E55TZT+eHBwWAH6xNLQ43bZLxrYbS9q1xcyCDGyHxpTHfSQ2+cvQzEihEFPZ/h2LW8zb4YFlPvUo9+bgL2G68wQhA7Ib559cSjfTgsK/jCGF1Q311O26mXD3y8vXvO0YalVcvfFjzv5WggzGAlce0r4neAgfUgXL4jmbrb1LoIA4spQ2lPoMohqsfac9uOAkApXiuTz/ablK4EAEDoCN4Z+dGy+KWQvJa/tRsI0MoPKP3+L7N33cUHkRmSKjP3XpGdtDfsRwf1XPQ=
```
Comment (Kuschel-Swein) [inline content/tasks/third/epilogue.md:1]: Check if that uses the correct heading level please

**Review by Kuschel-Swein — CHANGES_REQUESTED** [review]

File: content/tasks/second/epilogue.md (hunk 1 above)
Comment (MariusLang) [reply content/tasks/second/epilogue.md:1]: The epilogue of Task 2 uses a headline level 1, as does my section. Could you clarify in more detail what specifically is wrong in this context?

File: content/tasks/third/index.md (hunk 2 above)
Comment (MariusLang) [reply content/tasks/third/index.md:1]: The description of Task 1 and Task 2 uses a headline level 1, as does my section. Could you clarify in more detail what specifically is wrong in this context?

File: content/tasks/third/epilogue.md (hunk 3 above)
Comment (MariusLang) [reply content/tasks/third/epilogue.md:1]: Here a headline level 1 is used as well as in the over epilogues.

File: content/tasks/second/epilogue.md (hunk 1 above)
Comment (Kuschel-Swein) [reply content/tasks/second/epilogue.md:1]: I guess thats my bad.

**Review by Kuschel-Swein — APPROVED** [review]

**Pull request merged**